### PR TITLE
Update documentation for CircuitBreaker

### PR DESCRIPTION
**Pull request by helenmasters**
* Standardized README
 * Updated apidoc
 * Added usage examples to apidoc
 * Removed SwiftyJSON from README example

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,129 +1,148 @@
...
-breaker.run(commandArgs: id, fallbackArgs: "Something went wrong.")
+let requestParam: String = "myRequestParams"
+breaker.run(commandArgs: requestParam, fallbackArgs: "Something went wrong.")
 ```
 
-###### Full Implementation:
+#### Full Implementation
 
 ```swift
 ...
```
Comment (Andrew-Lees11) [inline README.md]: Did you mean to remove these three dots along with the others?

File: Sources/CircuitBreaker/CircuitBreaker.swift [hunk 2]
```diff
@@ -78,21 +100,27 @@ public class CircuitBreaker<A, B> {
 
   // MARK: Initializers
 
-  /// Initializes CircuitBreaker instance with asyncronous context command (Advanced usage)
+  /// Initializes a CircuitBreaker instance with an asyncronous context command. (Advanced usage.)
```
Comment (Andrew-Lees11) [inline Sources/CircuitBreaker/CircuitBreaker.swift]: I'm not sure what is meant by (Advanced usage) here. Is using this initializer advances?

File: Sources/CircuitBreaker/Invocation.swift [hunk 3]
```diff
@@ -70,21 +73,37 @@ public class Invocation<A, B> {
...
-  /// Marks invocation as having timed out.
+  /// Marks invocation as having timed out if and only if the execution
+  /// of the invocation has not completed yet. In such case, true is returned;
+  /// otherwise, false.
+  /// ### Usage Example: ###
+  /// ```swift
+  /// result = invocation.notifyTimedOut()
+  /// ```
+  public func notifyTimedOut() -> Bool {
+    let boolResult = nofityTimedOut()
```
Comment (Andrew-Lees11) [inline Sources/CircuitBreaker/Invocation.swift]: This could just be `return nofityTimedOut()`

File: README.md (hunk 1 above)
Comment (helenmasters) [reply README.md]: Removed as suggested.

File: Sources/CircuitBreaker/Invocation.swift (hunk 3 above)
Comment (helenmasters) [reply Sources/CircuitBreaker/Invocation.swift]: Good point - thanks I've changed this as suggested.

File: Sources/CircuitBreaker/CircuitBreaker.swift (hunk 2 above)
Comment (helenmasters) [reply Sources/CircuitBreaker/CircuitBreaker.swift]: I've removed the words "Advanced usage" as I think that makes it less confusing.

**Pull request merged**